files/build_templates/docker_image_ctl.j2

```diff
@@ -146,6 +146,8 @@ start() {
         -e PRM_SNIFFER \
         -e PRM_SNIFFER_FILE_PATH \
         -v /var/log/mellanox/sniffer:/var/log/mellanox/sniffer:rw \
+        -v sx_api:/tmp \
```

Comment:
>sx_api [](start = 11, length = 6)

How about name it as 'mellanox_volume', and keep all shared sockets there, including 'shm' ?

Reply:
I tried to make /dev/shm as a volume with "-v volumename:/dev/shm", but it can not work, no share memory files can be created under it, I searched and seems it related to the dedicated filesystem for it, "The POSIX shared memory object implementation on Linux 2.4 makes use of a dedicated filesystem, which is normally mounted under /dev/shm."  The only way I have found to make it exposed and work is to mount host /dev/shm to it: "-v /dev/shm:/dev/shm"

And besides above, can we map multi paths to a volume? given that in our SDK implementation the unix socket and shm are under different path.  I didn't find a way to make volume support sub dirs, like "-v volume/folders:/tmp" can not work, any comments?